cloud/amazon/ec2_vpc_route_table.py

```diff
@@ -465,17 +471,19 @@ def create_route_spec(connection, routes, vpc_id):
 def ensure_route_table_present(connection, module):
 
     lookup = module.params.get('lookup')
-    propagating_vgw_ids = module.params.get('propagating_vgw_ids', [])
+    propagating_vgw_ids = module.params.get('propagating_vgw_ids')
     route_table_id = module.params.get('route_table_id')
     subnets = module.params.get('subnets')
     tags = module.params.get('tags')
     vpc_id = module.params.get('vpc_id')
-    check_mode = module.params.get('check_mode')
     try:
         routes = create_route_spec(connection, module.params.get('routes'), vpc_id)
     except AnsibleIgwSearchException as e:
         module.fail_json(msg=e[0])
 
+    if not propagating_vgw_ids:
+        propagating_vgw_ids = []
+
```

Comment:
That explanation helps a ton @erydo.  I've pushed a commit that removes this regression.